src/server.c

```diff
@@ -5267,13 +5268,24 @@ void sendChildCOWInfo(int ptype, char *pname) {
     size_t private_dirty = zmalloc_get_private_dirty(-1);
 
     if (private_dirty) {
-        serverLog(LL_NOTICE,
+        serverLog(LL_VERBOSE,
             "%s: %zu MB of memory used by copy-on-write",
-            pname, private_dirty/(1024*1024));
+            pname, private_dirty);
+    }
+
+    sendChildInfo(ptype, private_dirty);
+}
+
+void sendLastChildCOWInfo(int ptype, char *pname) {
```

Comment:
since these two function are nearly identical, let's add a top comment or something that explains that the reason is the difference is the log level.
or better yet, let's unify them into one implementation that takes a loglevel as argument and two aliases that call it.